tests/Unit/Domain.LinnApps.Tests/PurchaseOrderServiceTests/WhenUpdatingPurchaseOrderConvValueQtyFields.cs

```diff
@@ -486,7 +486,7 @@ public void ShouldUpdateSecondDeliveryTotalFields()
             var firstDetail = this.current.Details.First();
 
             // get second delivery
-            var delivery = firstDetail.PurchaseDeliveries.Last();
+            var delivery = firstDetail.PurchaseDeliveries.ElementAt(1);
```

Comment:
I think you should revert this test and add another for the scenario in question. Ideally you should add the new test before making any code changes

Reply:
or perhaps two new tests in this scenario?

Reply:
That's fair, I just figured that it was an extension of the current functionality so could just add to the test but it's fine to add a completely new one.